tests/test_neuron_functional.py

```diff
@@ -294,11 +290,11 @@ def test_breaker_of_tmd_algo():
     assert_array_equal(N.apical_sections, [33])
     assert_array_almost_equal(
         n.sections[169].points[-1],
-        np.array([-220.93813, -21.49141, -55.93323]),
+        np.array([4.5475516, 46.869442, 49.396336]),
```

Comment:
Why changing the diameter model changes the result points? Is it because the diametrizer calls `np.random` and thus shifts the generated random numbers?

Reply:
ah yeah ok.
Well, it would be nice to find the right ID so we can see that this commit changes section IDs but not the points (the current state implies that the points are change when using the new diametrizer while it is not the case).

Reply:
I know, but I have no idea how to do that in a reasonable timeframe... if you have any idea, you're welcome!

Reply:
```python
    for num, sec in n.sections.items():
        if np.allclose(sec.points[-1], np.array([-220.93813, -21.49141, -55.93323]), atol=1e-3, rtol=1e-3):
            print(num)

    for num, sec in n.sections.items():
        if np.allclose(sec.points[-1], np.array([-17.31787, 151.4876, -6.67741]), atol=1e-3, rtol=1e-3):
            print(num)
```
did the trick ;-p

Reply:
you are the best! thanks!